kitchen/modules/weights/src/lib.rs

```diff
@@ -0,0 +1,184 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+// Transaction Weight Examples
+// https://crates.parity.io/sr_primitives/weights/index.html
+use support::{ensure, decl_module, decl_storage, dispatch::{Result, WeighData}};
+use runtime_primitives::weights::{ DispatchClass, Weight, ClassifyDispatch, SimpleDispatchInfo };
+
+pub trait Trait: system::Trait {}
+
+decl_storage! {
+	trait Store for Module<T: Trait> as SimpleMap {
+		StoredValue get(stored_value): u32;
+	}
+}
+
+/// A "scale" to weigh transactions. This scale can be used
```

Comment:
better to set your line width to 100 like substrate.

Reply:
I'd like to know more about headers. Do you have a link to read?

I do know if I remove this line, the build fails.